src/validation/validators/rules.js

```diff
@@ -0,0 +1,11 @@
+import { check } from 'express-validator';
+
+// add validation rules here.
+export const userRegister = [
+  check('firstName').matches(/^[\w'\-,.][^0-9_!¡?÷?¿/\\+=@#$%ˆ&*(){}|~<>;:[\]]{1,250}$/).withMessage('firstName is invalid, firstName should be an alphabetical character, greater then 1 and less than 250 characters, should not include numbers and cannot be left empty').trim(),
+  check('lastName').matches(/^[\w'\-,.][^0-9_!¡?÷?¿/\\+=@#$%ˆ&*(){}|~<>;:[\]]{1,250}$/).withMessage('lastName is invalid, lastName should be an alphabetical character, greater then 1 and less than 250 characters, should not include numbers and cannot be left empty').trim(),
```

Comment:
@Oliver-ke  include an explanation of the regex

Reply:
Ok @eokwukwe, I've done that, thanks for the feedback